client/client.go

```diff
@@ -179,8 +188,38 @@ func NewClient(config Config) (XOClient, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	var token string
+	err = c.Call(context.Background(), "token.create", map[string]interface{}{}, &token)
+	if err != nil {
+		return nil, err
+	}
+
+	jar, err := cookiejar.New(&cookiejar.Options{})
+	if err != nil {
+		return nil, err
+	}
+
+	restApiURL, err := url.Parse(strings.Replace(wsURL, "ws", "http", 1))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Does this only replace the first instance of `ws`, or all instances?

Reply:
Yep. I added a test to verify this in a later commit.